examples/libadwaita/Cargo.toml

```diff
@@ -8,6 +8,7 @@ edition = "2021"
 rand = "0.8.5"
 relm4 = { path = "../../relm4", features = ["libadwaita"] }
 tokio = { version = "1.25", features = ["time"] }
+adw = { version = "0.5", package = "libadwaita" }
```

Comment:
This is not needed btw. because Relm4 with the "libadwaita" feature re-exports it, so you can just use `use relm4::adw;` to have `adw` in scope.

Reply:
It's not added for the `asw` import, but for the `adw/v1_2` required feature in the example.

Reply:
Oh, I didn't see that. Yet, you could also use "relm4/gnome_43" instead, which will also enable the required libadwaita feature.

Reply:
I will do that